From: Rui Salvaterra <rsalvaterra@gmail.com>
Subject: sstrip: don't try to compile elftoc

We only need sstrip itself and elftoc doesn't compile with musl's elf.h, so
disable the elftoc compilation in the makefile.

Signed-off-by: Rui Salvaterra <rsalvaterra@gmail.com>
---

--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 
 prefix = /usr/local
 
-PROGRAMS = elfls objres rebind sstrip elftoc ebfc infect
+PROGRAMS = elfls objres rebind sstrip ebfc infect
 
 all: $(PROGRAMS)
 
@@ -19,7 +19,6 @@ elfls: bin/elfls doc/elfls.1
 objres: bin/objres doc/objres.1
 rebind: bin/rebind doc/rebind.1
 sstrip: bin/sstrip doc/sstrip.1
-elftoc: bin/elftoc doc/elftoc.1
 ebfc: bin/ebfc doc/ebfc.1
 infect: bin/infect doc/infect.1
 
